style.css

```diff
@@ -3,11 +3,13 @@
     padding: 0;
 }
 
-.ques-header {
+.quiz-wrapper {
     background: var(--c-lightblue);
+    height: 80vh;
```

Comment:
In height avoid using vh

Reply:
thanks @MayankKumar10. but vh is working fine for this use case as ill also be adding footer in the future.